### ISSUE TITLE
Exporting key error

### ISSUE THREAD

**Issue by allenzhao** [jorgelbg/pinentry-touchid#25]
I was trying to export my keys and found that it's not possible in the command line using gpg  --armor --export-secret-keys

I end up using something like this:
```
echo mypassphrase | gpg --output private_keys.backup --armor --export-secret-keys --passphrase-fd 0 --pinentry-mode loopback
```
Is this expected?

**Comment by jorgelbg** [comment]
I don't think that we currently support this use case. From a quick overview it seems that we would need to skip all the touchid integration (since you are directly providing the passphrase). I will try to take a closer look and see if we can automatically fallback to pinentry-mac in this case.
